<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sqGisMainWindow</class>
 <widget class="QMainWindow" name="sqGisMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sqGisTool</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>图层</string>
    </property>
    <addaction name="openVectorLayerAction"/>
    <addaction name="openRasterLayerAction"/>
    <addaction name="separator"/>
    <addaction name="openLocalTilesLayerAction"/>
    <addaction name="openOpenStreetMapLayerAction"/>
    <addaction name="separator"/>
    <addaction name="removeLayerAction"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="selectAction"/>
    <addaction name="zoomInAction"/>
    <addaction name="zoomOutAction"/>
    <addaction name="panAction"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="openLayerMgrAction"/>
    <addaction name="openLayerPropertyAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>标绘</string>
    </property>
    <addaction name="markPointAction"/>
    <addaction name="markLineAction"/>
    <addaction name="markPolygonAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="convertCoorAction"/>
    <addaction name="separator"/>
    <addaction name="navMsgReceiveAction"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>测量分析</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="separator"/>
    <addaction name="action_13"/>
    <addaction name="action_15"/>
    <addaction name="separator"/>
    <addaction name="action_17"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openVectorLayerAction"/>
   <addaction name="openRasterLayerAction"/>
   <addaction name="openLocalTilesLayerAction"/>
   <addaction name="openOpenStreetMapLayerAction"/>
   <addaction name="removeLayerAction"/>
   <addaction name="separator"/>
   <addaction name="selectAction"/>
   <addaction name="zoomInAction"/>
   <addaction name="zoomOutAction"/>
   <addaction name="panAction"/>
   <addaction name="separator"/>
   <addaction name="markPointAction"/>
   <addaction name="markLineAction"/>
   <addaction name="markPolygonAction"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="zoomInAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/zoom-in_x24</normaloff>:/img/zoom-in_x24</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/zoom-out_x24</normaloff>:/img/zoom-out_x24</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="panAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/pan_x24</normaloff>:/img/pan_x24</iconset>
   </property>
   <property name="text">
    <string>拖动</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>添加位置点</string>
   </property>
  </action>
  <action name="openLayerMgrAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/layer-mgr_x24</normaloff>:/img/layer-mgr_x24</iconset>
   </property>
   <property name="text">
    <string>图层管理</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>选择区域</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>还原</string>
   </property>
  </action>
  <action name="markPointAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/position_x24</normaloff>:/img/position_x24</iconset>
   </property>
   <property name="text">
    <string>地理点</string>
   </property>
  </action>
  <action name="markLineAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/line_x24</normaloff>:/img/line_x24</iconset>
   </property>
   <property name="text">
    <string>地理轨迹</string>
   </property>
  </action>
  <action name="markPolygonAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/polygon_x24</normaloff>:/img/polygon_x24</iconset>
   </property>
   <property name="text">
    <string>封闭区域</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>轨迹导入</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>轨迹导出</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>空间距离测量</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>投影距离测量</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>空间角度测量</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>投影角度测量</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>可视域分析</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于sqGis</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>新建矢量图层</string>
   </property>
  </action>
  <action name="openVectorLayerAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/layer-vector-add_x24</normaloff>:/img/layer-vector-add_x24</iconset>
   </property>
   <property name="text">
    <string>打开矢量地图文件</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>新建栅格图层</string>
   </property>
  </action>
  <action name="openRasterLayerAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/layer-raster-add_x24</normaloff>:/img/layer-raster-add_x24</iconset>
   </property>
   <property name="text">
    <string>打开栅格地图文件</string>
   </property>
  </action>
  <action name="openLayerPropertyAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/layer-info_x24</normaloff>:/img/layer-info_x24</iconset>
   </property>
   <property name="text">
    <string>图层属性</string>
   </property>
  </action>
  <action name="openLocalTilesLayerAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/tile_x24</normaloff>:/img/tile_x24</iconset>
   </property>
   <property name="text">
    <string>打开本地瓦片地图</string>
   </property>
  </action>
  <action name="openOpenStreetMapLayerAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/osm_x24</normaloff>:/img/osm_x24</iconset>
   </property>
   <property name="text">
    <string>打开OpenStreetMap在线地图</string>
   </property>
  </action>
  <action name="selectAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/select_x24</normaloff>:/img/select_x24</iconset>
   </property>
   <property name="text">
    <string>选择指针
</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>实时轨迹</string>
   </property>
  </action>
  <action name="convertCoorAction">
   <property name="text">
    <string>EPSG3857&lt;=&gt;EPSG4326</string>
   </property>
  </action>
  <action name="removeLayerAction">
   <property name="icon">
    <iconset resource="sqGis.qrc">
     <normaloff>:/img/trash_x24</normaloff>:/img/trash_x24</iconset>
   </property>
   <property name="text">
    <string>移除图层</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="navMsgReceiveAction">
   <property name="text">
    <string>导航电文接收器</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sqGis.qrc"/>
 </resources>
 <connections/>
</ui>
